### PR TITLE
241 consolidate data and app services

### PR DESCRIPTION
**Pull request by esheehan-gsl**

### CODE REVIEW

**Comment by ian-noaa** [comment]
This looks like a good change. A couple of thoughts:

1. AWS now has a Python 3.10 image available for Lambdas if we wanted to bump up to 3.10. https://docs.aws.amazon.com/lambda/latest/dg/python-image.html#python-image-base
2. We'll want to downgrade the application's Dockerfile to use Python 3.9/3.10 instead of the current 3.11.
3. I think the maximum Lambda image size is 10 GB (uncompressed) - if that does become an issue, looking into Poetry's dependency groups would make some sense.

File: .github/workflows/api.yaml [hunk 1]
```diff
@@ -86,7 +80,6 @@ jobs:
         uses: irongut/CodeCoverageSummary@v1.3.0
         with:
           header: API Coverage
-          filename: services/api/coverage.xml
```
Comment (ian-noaa) [inline .github/workflows/api.yaml:89]: It looks like `filename` is required by the [CodeCoverageSummary action](https://github.com/irongut/CodeCoverageSummary#inputs). So we'll probably need something like this for the action to run:

```yaml
        with:
          header: API Coverage
          filename: coverage.xml
```

Comment (esheehan-gsl) [reply .github/workflows/api.yaml:89]: Oh, oops. I deleted that by mistake. 🤦🏻

**Comment by esheehan-gsl** [comment]
@ian-noaa I think this is ready for review

**Review by ian-noaa — APPROVED** [review]
Looks good to me. Nice catch on the `CONTRIBUTING` readme. I always forget about that one.

File: .prettierignore [hunk 2]
```diff
@@ -1,3 +1,9 @@
+tmp/
+.github/
+
+# FIXME: Maybe we should lint/format the k8s files?
+kubernetes/
```
Comment (ian-noaa) [inline .prettierignore:5]: I'll spin that out into an issue. It'd be good to do.

Comment (ian-noaa) [reply .prettierignore:5]: We now have an issue for this here: https://github.com/NOAA-GSL/unified-graphics/issues/328

**Pull request merged**